astropy/table/table.py

```diff
@@ -150,6 +150,19 @@ def keys(self):
     def values(self):
         return list(OrderedDict.values(self))
 
+    def isinstance(self, *cls):
+        '''
+        Return a list of columns which are instances of the specified classes.
```

Comment:
Since this is a public method you need to have a full docstring with the parameters and return values fully specified.